### ISSUE TITLE
intercepts still causing problems in print_model_weights

### ISSUE THREAD

**Issue by aoifecahill** [EducationalTestingService/skll#322]
Trying to print model weights for a logistic regression model that had fit_intercept set to False gives the following error: 

```
Traceback (most recent call last):
  File "/tmp/conda_env/bin/print_model_weights", line 9, in <module>
    load_entry_point('skll==1.2', 'console_scripts', 'print_model_weights')()
  File "/Users/acahill/projects/skll/skll/utilities/print_model_weights.py", line 56, in main
    (weights, intercept) = learner.model_params
  File "/Users/acahill/projects/skll/skll/learner.py", line 726, in model_params
    if self.model.intercept_.any():
AttributeError: 'float' object has no attribute 'any'
```

**Comment by desilinguist** [comment]
I will take a look at this.

**Comment by desilinguist** [comment]
Addressed by #323.
